Library/Homebrew/sorbet/tapioca/compilers/tty.rb

```diff
@@ -0,0 +1,36 @@
+# typed: strict
+# frozen_string_literal: true
+
+require_relative "../../../global"
+require_relative "../../../utils/tty"
+
+module Tapioca
+  module Compilers
+    class Tty < Tapioca::Dsl::Compiler
+      # FIXME: Enable cop again when https://github.com/sorbet/sorbet/issues/3532 is fixed.
+      # rubocop:disable Style/MutableConstant
+      # This should be a module whose singleton class contains RuboCop::AST::NodePattern::Macros,
+      #   but I don't know how to express that in Sorbet.
+      ConstantType = type_member { { fixed: Module } }
+      # rubocop:enable Style/MutableConstant
+
+      sig { override.returns(T::Enumerable[Module]) }
+      def self.gather_constants
+        [::Tty]
+      end
```

Comment:
This looks nicer as an endless def, IMO:
```suggestion
      def self.gather_constants = [::Tty]
```